content/news/core-monthly/2023/core-monthly-2022-12-01-2022-12-31.md

```diff
@@ -0,0 +1,377 @@
+
+In 2022 the project welcomed many new members from the community to help it grow. It was also the following year when you could see other project members and me in the [Public Demo](/news/2022/upcoming-demo-9-2022/) and for the first time, discuss with us during the [office hours](/news/2022/office-hours/).
+
+All project members wish you all the best in 2023! Now, let's get into the summary of December.
+
+Here's what happened:
+
+### Release of PrestaShop 1.7.8.8
+
+PrestaShop 1.7.8.8 was released on December 8th. This maintenance release fixed some bugs and regressions but, more importantly, provided the community with a security fix. It's worth noting that it's the last regular version of PrestaShop 1.7. You can [read release notes for this version](/news/2022/prestashop-1-7-8-8-maintenance-release/) to know more.
+
+### PrestaShop 8.0.1
+
+The first patch version for PrestaShop 8 is on the horizon. With over [30 bug fixes](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Amerged+is%3Apr+milestone%3A8.0.1+label%3A%22Bug+fix%22) and [plenty of minor improvements](https://github.com/PrestaShop/PrestaShop/pulls?page=1&q=is%3Amerged+is%3Apr+milestone%3A8.0.1+label%3AImprovement) this version aims to increase the stability of the latest major release.
+
+I understand that some of you wait for the first patch to start thinking about the upgrade, the good news is that both, the autoupgrade module and the patch versions are very very close to being fully available.
+
+### Autoupgrade module
+
+Speaking of upgrades, [autoupgrade v4.15 is now live](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.15.0). By the time I wrote this article, PrestaShop 8 was not available in the module interface, but the module itself had everything needed for the upgrade to succeed. If you don’t see PrestaShop 8 as a new major release, you can upgrade to the latest version using the `Local archive` or `Local directory` option.
+
+The release, along with the .xml file required by the module, [is available on GitHub](https://github.com/PrestaShop/PrestaShop/releases/tag/8.0.0) and if you don’t know how to use these options, [you can see a short demo](https://youtu.be/SBz7lApJ_pA?t=1063) of the process that I showed during the Public Demo.
+
+### Pull requests overview
+
+In December, there were over 200 merged pull requests. Let's dig into some of the most interesting contributions that target PrestaShop 8.1 and module updates.
+
+**Enabling profiler in back office**
+
+Thanks to [@Hlavtox](https://github.com/Hlavtox), in PrestaShop 8.1, it will be possible to enable performance metrics in the front office directly from the back office. There's still some work to do regarding this feature, as the community feedback was to add an extra security layer to let only shop maintainers see the results of profiling. You can see the [details in pull request](https://github.com/PrestaShop/PrestaShop/pull/30603).
```

Comment:
```suggestion
Thanks to [@Hlavtox](https://github.com/Hlavtox), in PrestaShop 8.1, it will be possible to enable performance metrics in the front office directly from the back office. There's still some work to do regarding this feature, as the community feedback was to add an extra security layer to let only shop maintainers see the results of profiling. You can see the [details in the pull request](https://github.com/PrestaShop/PrestaShop/pull/30603).
```